extensions/cornerstone-dicom-seg/src/viewports/OHIFCornerstoneSEGViewport.tsx

```diff
@@ -1,18 +1,13 @@
-import React, { useCallback, useEffect, useRef, useState } from 'react';
-import PropTypes from 'prop-types';
 import OHIF, { utils } from '@ohif/core';
 import {
-  Notification,
-  ViewportActionBar,
-  useViewportGrid,
-  useViewportDialog,
-  LoadingIndicatorProgress,
+  LoadingIndicatorProgress, Notification, useViewportDialog, useViewportGrid, ViewportActionBar
 } from '@ohif/ui';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
having reeact as the first import matches our pattern to have external imports first

Reply:
Sure thing. I think this might be the result of using the the "imports cleanup" feature of VSCode.